jrnl/plugins/exporter/dates.py

```diff
@@ -7,7 +7,7 @@
 
 from jrnl.plugins.base import BaseExporter
 
-from jrnl.__version__ import __version__
+from ...__version__ import __version__
```

Comment:
Can you separate the relative imports here since there's already another PR for that change, please?

Reply:
Rebased to drop this commit.